runtime/src/lib.rs

```diff
@@ -4,17 +4,24 @@
 // `construct_runtime!` does a lot of recursion and requires us to increase the limit to 256.
 #![recursion_limit = "256"]
 
-// Make the WASM binary available.
+// Make the WASM_BINARY available, but hide WASM_BINARY_BLOATY.
 #[cfg(feature = "std")]
-include!(concat!(env!("OUT_DIR"), "/wasm_binary.rs"));
+mod wasm {
```

Comment:
That include! imports two definitions. WASM_BINARY_BLOATY is the runtime built as wasm. WASM_BINARY is the stripped version of WASM_BINARY_BLOATY. It's smaller so that's what we use on-chain. A smaller binary helps the extrinsic to upgrade runtime small.

The dance I'm doing here because I want to expose WASM_BINARY publicly, but not WASM_BINARY_BLOATY.